packages/core-components/README.md

```diff
@@ -0,0 +1,208 @@
+COMPONENTS
```

Comment:
This is great, I have added it to [OKTA-361478] (https://oktainc.atlassian.net/browse/OKTA-361478)